frontend/benefit/applicant/src/components/applications/forms/application/step2/ApplicationFormStep2.tsx

```diff
@@ -386,7 +386,8 @@ const ApplicationFormStep2: React.FC<DynamicFormStepComponentProps> = ({
           </SelectionGroup>
         </$GridCell>
 
-        {formik.values.benefitType === BENEFIT_TYPES.SALARY && (
+        {formik.values.benefitType === BENEFIT_TYPES.SALARY 
+          && formik.values.apprenticeshipProgram === true &&(
```

Comment:
It seems the formatter is not working
```suggestion
          && formik.values.apprenticeshipProgram === true && (
```